api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -0,0 +1,77 @@
+import random
+
+import factory.fuzzy
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Oubli ?